cmake/Dependencies.cmake

```diff
@@ -935,23 +935,18 @@ if(BUILD_PYTHON)
   # then these will just use "python", but at least they'll be consistent with
   # each other).
   if(NOT DEFINED PYTHON_INCLUDE_DIR)
-    # distutils.sysconfig, if it's installed, is more accurate than sysconfig,
-    # which sometimes outputs directories that do not exist
-    pycmd_no_exit(_py_inc _exitcode "from distutils import sysconfig; print(sysconfig.get_python_inc())")
+    # TODO: Verify that sysconfig isn't inaccurate
+    pycmd_no_exit(_py_inc _exitcode "import sysconfig; print(sysconfig.get_path('include'))")
```

Comment:
This is not the exactly the same on windows, before it is `C:\tools\miniconda3\include`  and after `C:\tools\miniconda3\Include` (`include` is the actual name of the folder), but since it's Windows it's case insensitive and doesn't matter